### PR TITLE
show accept/reject on submissions

### PR DESCRIPTION
**Pull request by apollo1291**
if and only if status="OK!":
then show "Accepted" or "Rejected" (based on the accepted var)

This solution works if the submission status is anything other than ok, I am not sure that it works if status is OK! because I don't know how to create a submission that is "OK!" to test it. I think the logic is right though.

### CODE REVIEW

File: frontend/src/components/submissionList.js [hunk 1]
```diff
@@ -35,6 +40,7 @@ class SubmissionList extends Component {
       // This check handles the case where a new page is requested while a
       // previous page was loading.
       if (page == this.state.page) {
+        console.log(data);
```
Comment (apollo1291) [inline frontend/src/components/submissionList.js:43]: will delete this once tested

File: frontend/src/components/submissionList.js [hunk 2]
```diff
@@ -50,7 +56,11 @@ class SubmissionList extends Component {
       return (
         <tr key={submission.id}>
           <td>{new Date(submission.created).toLocaleString()}</td>
-          <td>{SUBMISSION_STATUS[submission.status]} </td>
+          <td>
+            {submission.status == "OK!" &&
```
Comment (apollo1291) [inline frontend/src/components/submissionList.js:60]: is this logic consistent / is there a better way to do this

File: frontend/src/components/submissionList.js [hunk 3]
```diff
@@ -50,7 +56,11 @@ class SubmissionList extends Component {
...
+              SUBMISSION_ACCEPTED[submission.accecpted.toString()]}
+            {submission.status != "OK!" && SUBMISSION_STATUS[submission.status]}
+          </td>
```
Comment (apollo1291) [inline frontend/src/components/submissionList.js:63]: is this logic correct / is there a better way to do this?

File: frontend/src/components/submissionList.js [hunk 4]
```diff
@@ -50,7 +56,11 @@ class SubmissionList extends Component {
       return (
         <tr key={submission.id}>
           <td>{new Date(submission.created).toLocaleString()}</td>
-          <td>{SUBMISSION_STATUS[submission.status]} </td>
+          <td>
+            {submission.status == "OK!" &&
+              SUBMISSION_ACCEPTED[submission.accecpted.toString()]}
```
Comment (n8kim1) [inline frontend/src/components/submissionList.js:60]: typo in accepted

Comment (n8kim1) [inline frontend/src/components/submissionList.js:60]: also i don't think you need toString

**Review by n8kim1 — CHANGES_REQUESTED** [review]

**Review by n8kim1 — APPROVED** [review]

**Pull request merged**